gtcFuncs.py

```diff
@@ -133,6 +134,7 @@ def query():
     parser = argparse.ArgumentParser(description='Functions and methods for gtc files')
     parser.add_argument('method', choices=['manipulateGTCs', 'getIntensities', 'sampleInformation', 'createSampleSheet', 'allCombos'])
     parser.add_argument('--bpm', required=True, type=str, help='Full path to bead pool manifest file (.bpm); must be same one used to generate gtc')
+    parser.add_argument('--bpm-csv',required=True, type=str, help='Full path to bead pool manifest file in CSV form (.csv); must be same one used to generate gtc')
```

Comment:
Good idea -- I like this esp since Illumina does provide the csv and that also allows us to modify the csv if needed w/o going through the bpm